Resolve the JavaDoc warning of openmldb-jdbc

Now we can get logs of warning of Java module `openmldb-jdbc` when compiling.

```
10 warnings
[WARNING] Javadoc Warnings
[WARNING] /Users/tobe/code/4pd/OpenMLDB/java/openmldb-jdbc/src/main/java/com/_4paradigm/openmldb/jdbc/SQLResultSet.java:333: warning: no description
 for @param
[WARNING] * @param s
[WARNING] ^
[WARNING] /Users/tobe/code/4pd/OpenMLDB/java/openmldb-jdbc/src/main/java/com/_4paradigm/openmldb/common/LibraryLoader.java:93: warning: no @param fo
r path
[WARNING] public static String extractResource(String path, boolean isTemp) throws IOException {
[WARNING] ^
[WARNING] /Users/tobe/code/4pd/OpenMLDB/java/openmldb-jdbc/src/main/java/com/_4paradigm/openmldb/common/LibraryLoader.java:93: warning: no @param fo
r isTemp
[WARNING] public static String extractResource(String path, boolean isTemp) throws IOException {
[WARNING] ^
[WARNING] /Users/tobe/code/4pd/OpenMLDB/java/openmldb-jdbc/src/main/java/com/_4paradigm/openmldb/common/LibraryLoader.java:93: warning: no @return
[WARNING] public static String extractResource(String path, boolean isTemp) throws IOException {
[WARNING] ^
[WARNING] /Users/tobe/code/4pd/OpenMLDB/java/openmldb-jdbc/src/main/java/com/_4paradigm/openmldb/common/LibraryLoader.java:93: warning: no @throws f
or java.io.IOException
[WARNING] public static String extractResource(String path, boolean isTemp) throws IOException {
[WARNING] ^
[WARNING] /Users/tobe/code/4pd/OpenMLDB/java/openmldb-jdbc/src/main/java/com/_4paradigm/openmldb/sdk/QueryFuture.java:73: warning: no description fo
r @param
[WARNING] * @param timeUnit
[WARNING] ^
[WARNING] /Users/tobe/code/4pd/OpenMLDB/java/openmldb-jdbc/src/main/java/com/_4paradigm/openmldb/sdk/QueryFuture.java:74: warning: no description fo
r @return
[WARNING] * @return
[WARNING] ^
[WARNING] /Users/tobe/code/4pd/OpenMLDB/java/openmldb-jdbc/src/main/java/com/_4paradigm/openmldb/sdk/QueryFuture.java:75: warning: no description fo
r @throws
[WARNING] * @throws InterruptedException
[WARNING] ^
[WARNING] /Users/tobe/code/4pd/OpenMLDB/java/openmldb-jdbc/src/main/java/com/_4paradigm/openmldb/sdk/QueryFuture.java:76: warning: no description fo
r @throws
[WARNING] * @throws ExecutionException
[WARNING] ^
[WARNING] /Users/tobe/code/4pd/OpenMLDB/java/openmldb-jdbc/src/main/java/com/_4paradigm/openmldb/sdk/QueryFuture.java:77: warning: no description fo
r @throws
......
```

You can reproduce the issue by compiling java modules with `mvn clean package`.

--- a/java/openmldb-jdbc/src/main/java/com/_4paradigm/openmldb/common/LibraryLoader.java
+++ b/java/openmldb-jdbc/src/main/java/com/_4paradigm/openmldb/common/LibraryLoader.java
@@ -91,7 +91,7 @@ public class LibraryLoader {
     /**
      * Extract library in resource into filesystem
      * @param path the path that will be extracted.
-     * @param isTemp true if a temp file is created, otherwise false.
+     * @param isTemp whether to create a temp file.
      * @return the absolute path of the resources.
      * @throws IOException if the resources cannot be found.
      */

--- a/java/openmldb-jdbc/src/main/java/com/_4paradigm/openmldb/common/LibraryLoader.java
+++ b/java/openmldb-jdbc/src/main/java/com/_4paradigm/openmldb/common/LibraryLoader.java
@@ -87,8 +87,13 @@ public class LibraryLoader {
         }
     }
 
+
     /**
      * Extract library in resource into filesystem
+     * @param path the path that will be extracted.
+     * @param isTemp true if a temp file is created, otherwise false.
+     * @return the absolute path of the resources.
+     * @throws IOException if the resources cannot be found.
      */
     public static String extractResource(String path, boolean isTemp) throws IOException {
         InputStream inputStream = LibraryLoader.class.getClassLoader().getResourceAsStream(path);

--- a/java/openmldb-jdbc/src/main/java/com/_4paradigm/openmldb/jdbc/SQLResultSet.java
+++ b/java/openmldb-jdbc/src/main/java/com/_4paradigm/openmldb/jdbc/SQLResultSet.java
@@ -330,7 +330,7 @@ public class SQLResultSet implements ResultSet {
     }
 
     /**
-     * @param s
+     * @param s the string for the unicode stream extraction.
      * @deprecated
      */
     @Override

--- a/java/openmldb-jdbc/src/main/java/com/_4paradigm/openmldb/sdk/QueryFuture.java
+++ b/java/openmldb-jdbc/src/main/java/com/_4paradigm/openmldb/sdk/QueryFuture.java
@@ -70,11 +70,11 @@ public class QueryFuture implements Future<java.sql.ResultSet>{
     /**
      *
      * @param l  current timeout set by executeQeuryAsyn, so the param is invalid
-     * @param timeUnit
-     * @return
-     * @throws InterruptedException
-     * @throws ExecutionException
-     * @throws TimeoutException
+     * @param timeUnit the time unit of the timeout, which is also invalid.
+     * @return the result of the query from the database
+     * @throws InterruptedException throws when a thread is waiting, sleeping, or otherwise occupied, and the thread is interrupted, either before or during the activity.
+     * @throws ExecutionException throws when attempting to retrieve the result of a task that aborted by throwing an exception.
+     * @throws TimeoutException throws when a blocking operation times out.
      */
     @Override
     @Deprecated

